task 2/task2/lib/main.dart

```diff
@@ -0,0 +1,196 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'signup.dart';
+void main() => runApp(new MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      debugShowCheckedModeBanner: false,
+     routes: <String, WidgetBuilder>{
+        '/signup': (BuildContext context) => new SignupPage()
+      },
+      home: new MyHomePage(),
+    );
+  }
+}
+
+
+
+class MyHomePage extends StatefulWidget {
+  @override
+  _MyHomePageState createState() => new _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      resizeToAvoidBottomPadding: false,
+      body: Column(crossAxisAlignment: CrossAxisAlignment.start, children: <
+          Widget>[
+        Container(
+          child: Stack(
+            children: <Widget>[
```

Comment:
Remove the `Stack` since you are already using the `Column`. The `Stack` is used when you need to stack the widgets. In the current UI you are not stacking widgets.

Reply:
Removed stack!